recipes/ocaml/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+export LC_ALL=C
+./configure -prefix $PREFIX
+make world.opt
+umask 022
```

Comment:
Can we specify the file here which you want to target?

Reply:
This was copied from their instructions which didn't say. It may not be needed...